paddle/fluid/operators/slice_op.cc

```diff
@@ -155,7 +155,14 @@ class SliceOp : public framework::OperatorWithKernel {
 #ifdef PADDLE_WITH_MKLDNN
       auto input_data_type =
           framework::OperatorWithKernel::IndicateVarDataType(ctx, "Input");
-
+      auto vec_dims = phi::vectorize(in_tensor.dims());
+      bool all_zeros = true;
+      for (auto &dim : vec_dims) {
+        if (dim != 0) {
+          all_zeros = false;
+          break;
+        }
+      }
```

Comment:
Could you use `std::any_of` for it?
reference: https://en.cppreference.com/w/cpp/algorithm/all_any_none_of

Reply:
Thanks, done.